docs/user/pings/deletion_request.md

```diff
@@ -9,14 +9,36 @@ This ping contains the client id.
 This ping is intended to communicate to the Data Pipeline that the user wishes to have their reported Telemetry data deleted.
 As such it attempts to send itself at the moment the user opts out of data collection, and continues to try and send itself.
 
+> **Note:** It is possible to send secondary ids in the deletion request ping.  For instance, if the application is migrating
+> from legacy telemetry to Glean, the legacy client ids can be added to the deletion request ping by creating a `metrics.yaml`
+> entry for the id to be added with a `send_in_pings` value of `deletion_request`.
+>
+> An example `metrics.yaml` entry might look like this:
+> ```
+> legacy_client_id:
+>    type: uuid
+>    description:
+>      A UUID uniquely identifying the legacy client.
+>    send_in_pings:
+>      - deletion_request
+>    lifetime: user
+>    bugs:
+>      - <relevant bug url>
+>    data_reviews:
+>      - <relevant data review url>
+>    notification_emails:
+>      - product-team@example.com
+>    expires: never
+> ```
+
 ## Scheduling
 
 The `deletion-request` ping is automatically submitted when upload is disabled in Glean.
 If upload fails, it is retried after Glean is initialized.
 
 ## Contents
 
-The `deletion-request` does not contain additional metrics.
+The `deletion-request` does not contain additional metrics aside from secondary ids that have been added.
```

Comment:
Instead of having this content (bound to be outdated!), what if we link to [our autogenerated docs](https://mozilla.github.io/glean/book/user/collected-metrics/metrics.html#deletion-request)?

Reply:
Hmmm, sounds good to me.  It still says "This ping contains no metrics." which wouldn't necessarily be true if they added a legacy telemetry id to the ping.  I don't see that text in the `pings.yaml` file, how does it make it into the autogenerated docs?

Reply:
Mh, good point. Forget this one and leave your change as it is.